addons/mail/static/src/utils/common/format.js

```diff
@@ -225,6 +230,16 @@ async function _generateEmojisOnHtml(htmlString) {
     return htmlString;
 }
 
+/**
+ * @private
+ * @param {string} htmlString
+ * @returns {string}
+ */
+function generateEditLabel(htmlString) {
+    const editLabel = markup(_t(`<em class="o-mail-edit-label"> (edited)</em>`));
```

Comment:
There's no need to expose `<em class="o-mail-edit-label">` to translators, only `(edited)` is good.

Reply:
This should be added in python instead, at least in the update content route, and possibly even better as a write override when changing body.

This prevents user from being able to control id/remove it by making RPC manually.